awx_collection/plugins/modules/import.py

```diff
@@ -56,8 +56,6 @@
 
 # In this module we don't use EXPORTABLE_RESOURCES, we just want to validate that our installed awxkit has import/export
 try:
-    from awxkit.api.pages.api import EXPORTABLE_RESOURCES  # noqa
-
```

Comment:
post-change, this doesn't make any sense.

Reply:
should we just drop this try except block? do we really need to check whether awxkit has the import/export modules? I think that feature was long enough ago we can assume it is there